airbyte-container-orchestrator/src/test/resources/files/application.txt

```diff
@@ -0,0 +1 @@
+normalization-orchestrator
```

Comment:
should this say container-orchestrator instead?

Reply:
no, this file tells the container-orchestrator what kind of orchestrator to spin-up

Reply:
because we spin up the normalization orchestrator in tests?